readme.md

```diff
@@ -53,98 +50,64 @@ Broken into a variety of functional areas, the eligibility criteria is a checkli
+- The project is aligned with the philosophy and guidelines for collaborative development.
+- The project is built on the .NET platform and/or creates value within the .NET ecosystem.
+- The project solves a large enough problem and has a clear technology scope and focus which provides a reasonable likelihood of success.
 
 **Code**
 
--The project produces software for distribution to the public at no charge.
-
--The project's code is easily discoverable and publicly accessible (preferably on GitHub). 
-
--The project contains a build script that can produce deployable artifacts that are identical to the official deployable artifacts, with the exception of code signing (Exception may be granted for strong name keys, though strongly encouraged to be committed. Exception relies on OSS signing being in the build script for public builds).
-
--When applicable, project must use reproducible build settings in its toolchain.
-
--The project uses [Source Link](https://docs.microsoft.com/en-us/dotnet/standard/library-guidance/sourcelink).
-
--The project uses either embedded PDBs or publish symbol packages to NuGet (if applicable).
-
--The project code signs their artifacts as appropriate.
+- The project produces software for distribution to the public at no charge.
+- The project's code is easily discoverable and publicly accessible (preferably on GitHub). 
+- The project contains a build script that can produce deployable artifacts that are identical to the official deployable artifacts, with the exception of code signing (Exception may be granted for strong name keys, though strongly encouraged to be committed. Exception relies on OSS signing being in the build script for public builds).
+- When applicable, project must use reproducible build settings in its toolchain.
+- The project uses [Source Link](https://docs.microsoft.com/en-us/dotnet/standard/library-guidance/sourcelink).
+- The project uses either embedded PDBs or publish symbol packages to NuGet (if applicable).
+- The project code signs their artifacts as appropriate.
 
 **Licenses and Copyright**
 
--The code is offered under a standard, permissive open source license which has been approved by the .NET Foundation.
-
--Libraries that are mandatory dependencies of the project are also offered under a standard, permissive open source license which has been approved by the .NET Foundation (exceptions include a dependency on .NET Framework).
-
--Committers are bound by a Contributor License Agreement.
-
--The copyright ownership of everything that the project produces is clearly defined and documented.
+- The code is offered under a standard, permissive open source license which has been approved by the .NET Foundation.
+- Libraries that are mandatory dependencies of the project are also offered under a standard, permissive open source license which has been approved by the .NET Foundation (exceptions include a dependency on .NET Framework).
```

Comment:
What about the situation where a project targets proprietary hardware and such hardware requires a dependency on a proprietary license?

Reply:
See also https://github.com/dotnet-foundation/projects/pull/54#discussion_r382249773

Reply:
Maybe I'm being dense, but I didn't see how https://github.com/dotnet-foundation/projects/pull/54#discussion_r382249773 addressed my concern here.

Imagine this scenario: You have a .NET Lib that makes it easy to add hand gestures for the Oculus Rift. But it turns out you have to take a mandatory dependency on _oculus-rift.dll_ which is proprietary. The way this is worded right now, we would not allow such a project.

I think we should. I think we can make this a bit more generic.

```suggestion
- Libraries that are mandatory dependencies of the project are also offered under a standard, permissive open source license which has been approved by the .NET Foundation (exceptions include a dependency that is required by the target platform where no alternative open source dependency is available such as the .NET Framework or a hardware specific library).
```